components/integrations_marketplace/marketplace_modal.scss

```diff
@@ -0,0 +1,133 @@
+@import 'compass/utilities';
+@import 'compass/css3';
+@import 'utils/variables';
+
+.modal-marketplace {
+    display: flex;
+    flex-direction: column;
+    padding-top: 50px;
+    width: 100%;
+    height: 100%;
+    align-items: center;
+    font-size: 16px;
+    color: var(--center-channel-color-90);
+
+    @media (max-width: 768px) {
+        padding-left: 15px;
+        padding-right: 15px;
+    }
+
+    .nav-tabs {
+        font-size: 14px;
+
+        > li {
+            > a {
+                @include single-transition(all, .15s, ease);
+                padding: 10px 16px;
+            }
+        }
+    }
+
+    h1 {
+        font-size: 28px;
+        margin: 8px 0 24px;
+
+        strong {
+            font-weight: 400;
+        }
+    }
+
+    h2 {
+        font-weight: 300;
+        strong {
+            font-weight: 400;
+        }
+    }
+
+    .search_input {
```

Comment:
minor nit: `.search-input` or `.search__input`?

Reply:
This was the conventions that was used on other `.scss` files in the webapp. @asaadmahmood maybe has more insight on what convention we should follow here.